docs/releases/ibexa_dxp_v3.2.md

```diff
@@ -68,6 +68,12 @@ and [GraphQL](../api/graphql.md#jwt-authentication).
 
 See [JWT authentication](../guide/security.md#jwt-authentication) to learn how to configure this authentication method.
 
+### Searching in [[= product_name_com =]] with Elasticsearch
```

Comment:
We should spell our product name in release notes, as they are merged to all versions and it will cause problems with future variable changes.
```suggestion
### Searching in Ibexa Commerce with Elasticsearch
```